tools/build.sh

```diff
@@ -40,20 +60,28 @@ replace_icon_sprite_url() {
     OLD_PATH="\.\.\/\.\.\/\.\.\/node_modules\/@nimiq\/style\/nimiq-style.icons.svg"
     NEW_PATH="\/assets\/nimiq-style.icons.svg"
 
-    sed -i -e "s/$OLD_PATH/$NEW_PATH/g" $1
+    inplace_sed "s/$OLD_PATH/$NEW_PATH/g" $1
 }
 
 # replace font url in file $1
 replace_font_url() {
     OLD_PATH="(\.\.\/)*assets\/fonts"
     NEW_PATH="\/assets\/fonts"
 
-    sed -i -r -e "s/$OLD_PATH/$NEW_PATH/g" $1
+    inplace_sed -E "s/$OLD_PATH/$NEW_PATH/g" $1
+}
+
+# Replace xxd -r -p with a nice bash function
+function hex_to_binary() {
+  for (( i=0; i<${#1}; i+=2 ))
+  do
+      printf "\x${1:$i:2}"
+  done
 }
 
 # generate a base64 file integrity hash
 make_file_hash() {
-    echo $(${SHA256SUM} "$1" | awk '{print $1}' | xxd -r -p | base64)
+  hex_to_binary "$(${SHA256SUM} "$1" | cut -d' ' -f1)" | base64
```

Comment:
Would it be possible to make the output non-implicit? E.g. make it so that `echo` is still required here to return the value from `make_file_hash`?
Also, indentation is off by a space here.

Reply:
But why? Is there a reason? :) There isn't a difference. That's simply inefficient and unnecessary... (And frankly speaking, it also looks really painful for a BASH programmer's eye.)

Reply:
Indentation is fixed. Also for the `hex_to_binary` function.